### PR TITLE
PLA-268 Add docs pages for matview CRUD endpoints

### PR DESCRIPTION
**Pull request by TheEdgeOfRage**

### CODE REVIEW

File: api-reference/queries/endpoint/create-matview.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: 'Upsert Materialized View'
+openapi: 'POST /v1/materialized-views'
+---
+
+
+<Note>
+- This endpoint will create a new matview if none exists for the gven query ID, or update an existing one if the query ID matches.
+- It will fail to create a matview with the same name as an existing one, but for a different query ID
+- The name of the matview is just the last part of the matview, not including `dune.<your_team>.` and must be prefixed with `result_`
```
Comment (mewwts) [inline api-reference/queries/endpoint/create-matview.mdx:10]: this seems very user-unfriendly :p

File: api-reference/queries/endpoint/create-matview.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- The `cron_expression` parameter must be passed in as a valid 5 section cron expression. [See here](https://crontab.guru/) for how to write them.
```
Comment (mewwts) [inline api-reference/queries/endpoint/create-matview.mdx:11]: maybe add an example here or in the spec

File: api-reference/queries/endpoint/create-matview.mdx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+</Note>
+
+<Tip>
+- The cron expression interval must be at least 15 minutes and at most weekly
```
Comment (mewwts) [inline api-reference/queries/endpoint/create-matview.mdx:15]: isn't it "at most 15 minutes" and "at least weekly"? 🤔

File: api-reference/queries/endpoint/delete-matview.mdx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: 'Delete Materialized View'
+openapi: 'DELETE /v1/materialized-views/{name}'
+---
+
```
Comment (mewwts) [inline api-reference/queries/endpoint/delete-matview.mdx:5]: "this deletes the materialized view"

File: api-reference/queries/endpoint/list-matviews.mdx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+title: 'List Materialized Views'
+openapi: 'GET /v1/materialized-views'
+---
+
```
Comment (mewwts) [inline api-reference/queries/endpoint/list-matviews.mdx:5]: show how to use this with get?

File: api-reference/queries/endpoint/get-matview.mdx [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
+title: 'Get Materialized View'
+openapi: 'GET /v1/materialized-views/{name}'
+---
+
```
Comment (mewwts) [inline api-reference/queries/endpoint/get-matview.mdx:5]: show how to use this with list?

**Review by mewwts — APPROVED** [review]
lgtm but I have suggestions

File: api-reference/queries/endpoint/create-matview.mdx (hunk 1 above)
Comment (TheEdgeOfRage) [reply api-reference/queries/endpoint/create-matview.mdx:10]: This is a limitation that was added when we implemented matviews. I could add the prefix automatically if the user doesn't, that would make it a lot nicer :)

**Pull request merged**